### PR TITLE
refactor: rename prices -> exchange rates

### PR DESCRIPTION
**Pull request by sabineschaller**
## Changes proposed in this pull request

- rename prices -> exchange rates because the prices term was confusing
- add OpenAPI spec for exchange rates

## Context

- part of #626 

## Checklist

- [x] Related issues linked using `fixes #number`
- [x] Tests added/updated
- [x] Documentation added
- [x] Make sure that all checks pass

### CODE REVIEW

File: localenv/cloud-nine-wallet/docker-compose.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ services:
       PUBLIC_HOST: http://cloud-nine-wallet-backend
       OPEN_PAYMENTS_URL: http://cloud-nine-wallet-backend
       WEBHOOK_URL: http://cloud-nine-wallet/webhooks
-      PRICES_URL: http://cloud-nine-wallet/prices
+      EXCHANGE_RATES_URL: http://cloud-nine-wallet/prices
```
Comment (mkurapov) [inline localenv/cloud-nine-wallet/docker-compose.yml:56]: should we rename the `/prices` route in the ASE?

File: packages/backend/src/openapi/exchange-rates.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+openapi: 3.1.0
```
Comment (mkurapov) [inline packages/backend/src/openapi/exchange-rates.yaml:1]: will validation using this spec be added later when receiving the response from the /rates endpoint?

File: packages/backend/src/rates/service.ts [hunk 3]
```diff
@@ -96,23 +96,23 @@ class RatesServiceImpl implements RatesService {
...
     }
-    return this.pricesRequest
+    return this.ratesRequest
   }
 
-  private async loadNow(): Promise<Prices> {
-    const url = this.deps.pricesUrl
+  private async loadNow(): Promise<Rates> {
+    const url = this.deps.exchangeRatesUrl
     if (!url) return {}
```
Comment (mkurapov) [inline packages/backend/src/rates/service.ts:113]: when no url, should we return 
```ts
    const data = {
      [base]: 1.0,
      rates: {} 
    }
```
?

**Review by mkurapov — COMMENTED** [review]
a few comments

File: packages/backend/src/rates/service.ts (hunk 3 above)
Comment (raducristianpopa) [reply packages/backend/src/rates/service.ts:113]: Shouldn't it be just:
```ts
const data = {
  [base]: 1.0,
}
```
?

File: packages/backend/src/openapi/exchange-rates.yaml (hunk 2 above)
Comment (sabineschaller) [reply packages/backend/src/openapi/exchange-rates.yaml:1]: I started adding it but then I realized that responses are already validated. See [here](https://github.com/interledger/rafiki/blob/main/packages/backend/src/rates/service.ts#L115). Hence, the spec is rather for documentation than for validation.

File: packages/backend/src/rates/service.ts (hunk 3 above)
Comment (sabineschaller) [reply packages/backend/src/rates/service.ts:113]: @raducristianpopa is right, but it is not necessary. If `{}` is returned, `{ [base]: 1.0 }` is assumed by the pay library (which I think is the only code that consumes that response, but I'm still digging through the code to understand that part better).

File: localenv/cloud-nine-wallet/docker-compose.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (mkurapov) [reply localenv/cloud-nine-wallet/docker-compose.yml:56]: I think just to be consistent (to have the MASE as a good example) IMO it's worth it

[the remark this answers is not in the document]
Comment (mkurapov) [reply localenv/cloud-nine-wallet/docker-compose.yml:56]: I think just to be consistent (to have the MASE as a good example) IMO it's worth it

**Review by mkurapov — COMMENTED** [review]
Just the MASE change, otherwise LGTM

**Review by mkurapov — APPROVED** [review]

**Pull request merged**